pkg/apis/componentconfig/types.go

```diff
@@ -219,116 +234,144 @@ type KubeControllerManagerConfiguration struct {
 	// allowed to sync concurrently. Larger number = more responsive jobs,
 	// but more CPU (and network) load.
 	ConcurrentJobSyncs int32
-	// concurrentNamespaceSyncs is the number of namespace objects that are
-	// allowed to sync concurrently.
-	ConcurrentNamespaceSyncs int32
-	// concurrentSATokenSyncs is the number of service account token syncing operations
-	// that will be done concurrently.
-	ConcurrentSATokenSyncs int32
-	// lookupCacheSizeForRC is the size of lookup cache for replication controllers.
-	// Larger number = more responsive replica management, but more MEM load.
 	// nodeSyncPeriod is the period for syncing nodes from cloudprovider. Longer
 	// periods will result in fewer calls to cloud provider, but may delay addition
 	// of new nodes to cluster.
 	NodeSyncPeriod metav1.Duration
```

Comment:
this should be part of the KubeCloudShared, as it is used by multiple kube- and cloud-controllers.

Reply:
Done. Now that it Mark as Deprecated, we can move it to KubeCloudShared.